src/tech/v3/libs/xgboost.clj

```diff
@@ -182,9 +182,9 @@
    :round (ml-gs/linear 5 46 5 :int64)
    :alpha (ml-gs/linear 0.01 0.31 30)})
 
-(defn ->dmatrix [feature-ds target-ds sparse-column]
+(defn ->dmatrix [feature-ds target-ds sparse-column n-sparse-columns]
```

Comment:
allowing to parse the n-sparse-columns by user fixes similar issue in xboost then i fixed #37